### PR TITLE
optimize restlight.server.biz-threads.xxx configuration

### PR DESCRIPTION
**Pull request by Mkabaka**

### CODE REVIEW

**Comment by alalag1** [comment]
@Mkabaka plz use `restlight.server.biz-threads.xxx` instead of `restlight.server.biz-threads-xxx`

File: restlight-server/src/test/java/esa/restlight/server/RestliteTest.java [hunk 1]
```diff
@@ -89,9 +93,9 @@ void testDeployments() {
         assertTrue(biz instanceof ExecutorScheduler);
         assertTrue(((ExecutorScheduler) biz).executor() instanceof ThreadPoolExecutor);
         final ThreadPoolExecutor bizPool = (ThreadPoolExecutor) ((ExecutorScheduler) biz).executor();
-        assertEquals(ops.getBizThreadsCore(), bizPool.getCorePoolSize());
-        assertEquals(ops.getBizThreadsMax(), bizPool.getMaximumPoolSize());
-        assertEquals(ops.getBizThreadsKeepAliveTimeSeconds(), bizPool.getKeepAliveTime(TimeUnit.SECONDS));
```
Comment (alalag1) [inline restlight-server/src/test/java/esa/restlight/server/RestliteTest.java]: plz do not remove existing unit tests of deprecated functions

File: restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java [hunk 2]
```diff
@@ -49,16 +56,17 @@ void testConfigure() {
...
                 .configured();
 
         assertTrue(options.isHttp2Enable());
         assertTrue(options.isUseNativeTransports());
         assertEquals(2, options.getConnectorThreads());
         assertEquals(3, options.getIoThreads());
-        assertEquals(4, options.getBizThreadsCore());
-        assertEquals(5, options.getBizThreadsMax());
-        assertEquals(6, options.getBizThreadsBlockingQueueLength());
-        assertEquals(7, options.getBizThreadsKeepAliveTimeSeconds());
```
Comment (alalag1) [inline restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java]: plz do not remove existing unit tests of deprecated functions

File: restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java [hunk 3]
```diff
@@ -86,10 +94,6 @@ void testDefaultOpts() {
         assertEquals(def.isUseNativeTransports(), options.isUseNativeTransports());
         assertEquals(def.getConnectorThreads(), options.getConnectorThreads());
         assertEquals(def.getIoThreads(), options.getIoThreads());
-        assertEquals(def.getBizThreadsCore(), options.getBizThreadsCore());
-        assertEquals(def.getBizThreadsMax(), options.getBizThreadsMax());
-        assertEquals(def.getBizThreadsBlockingQueueLength(), options.getBizThreadsBlockingQueueLength());
-        assertEquals(def.getBizThreadsKeepAliveTimeSeconds(), options.getBizThreadsKeepAliveTimeSeconds());
```
Comment (alalag1) [inline restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java]: plz do not remove existing unit tests of deprecated functions

File: restlight-starter-actuator/src/main/java/esa/restlight/starter/actuator/autoconfigurer/ManagementOptions.java [hunk 4]
```diff
@@ -56,10 +58,7 @@ public String toString() {
         sb.append(", useNativeTransports=").append(isUseNativeTransports());
         sb.append(", connectorThreads=").append(getConnectorThreads());
         sb.append(", ioThreads=").append(getIoThreads());
-        sb.append(", bizThreadsCore=").append(getBizThreadsCore());
-        sb.append(", bizThreadsMax=").append(getBizThreadsMax());
-        sb.append(", bizThreadsBlockingQueueLength=").append(getBizThreadsBlockingQueueLength());
-        sb.append(", bizThreadsKeepAliveTimeSeconds=").append(getBizThreadsKeepAliveTimeSeconds());
```
Comment (alalag1) [inline restlight-starter-actuator/src/main/java/esa/restlight/starter/actuator/autoconfigurer/ManagementOptions.java]: plz do not remove existing unit tests of deprecated functions

**Review by alalag1 — CHANGES_REQUESTED** [review]
optimize unit tests plz

File: restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java (hunk 2 above)
Comment (Mkabaka) [reply restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java]: thx, have reverted!

File: restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java (hunk 3 above)
Comment (Mkabaka) [reply restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java]: thx, have reverted!

**Pull request merged**